GANDLF/compute/training_loop.py

```diff
@@ -217,24 +215,24 @@ def train_network(model, train_dataloader, optimizer, params):
 
 
 def training_loop(
-    training_data,
-    validation_data,
-    device,
-    params,
-    output_dir,
-    testing_data=None,
-    epochs=None,
-):
+    training_data: pd.DataFrame,
+    validation_data: pd.DataFrame,
+    device: str,
+    params: dict,
+    output_dir: str,
+    testing_data: bool = None,
+    epochs: bool = None,
```

Comment:
Same here withe epochs -it should be Union[int,None]

Reply:
Fixed with `Optional`.